types/k6/tsconfig.json

```diff
@@ -32,6 +32,8 @@
         "crypto/index.d.ts",
         "browser/index.d.ts",
 
+        "experimental/browser/index.d.ts",
```

Comment:
We removed the experimental browser module in [this PR](https://github.com/grafana/k6-DefinitelyTyped/pull/63).

Reply:
Sorry for the rebase mishap, fixing it 👍🏻